package.json

```diff
@@ -14,7 +14,7 @@
 		"node": ">=10"
 	},
 	"scripts": {
-		"test": "xo && nyc ava",
+		"test": "xo && nyc --reporter=lcov ava",
```

Comment:
Can you specify this as `nyc` config (in package.json) instead of a CLI argument?